shell/renderer/electron_render_frame_observer.cc

```diff
@@ -56,6 +57,8 @@ ElectronRenderFrameObserver::ElectronRenderFrameObserver(
       renderer_client_(renderer_client) {
   // Initialise resource for directory listing.
   net::NetModule::SetResourceProvider(NetResourceProvider);
+
+  render_frame_->GetWebView()->SetSupportsAppRegion(true);
```

Comment:
We should move this to main process and change similar to https://chromium-review.googlesource.com/c/chromium/src/+/5179536/10/chrome/browser/ui/web_applications/app_browser_controller.cc based on `PrimaryPageChanged`. Also based on this on this comment `// App regions are only supported in the main frame.` the change should be restricted to main frames.